CHANGELOG.md

```diff
@@ -114,7 +117,7 @@
 * (bluefox & AlCalzone) Add email address to MIT license (#232)
 
 ## v1.15.1 (2019-06-07)
-* (AlCalzone) Properly fix build breaks from #193.  
+* (AlCalzone) Properly fix build breaks from #193.
```

Comment:
Not sure how you have your editor set up, but please don't remove trailing whitespaces in Markdown.

Reply:
Oh sorry, that was not intended.
I've turned it off now for markdown files.